.github/workflows/release.yml

```diff
@@ -0,0 +1,203 @@
+        id: next_version
+        run: |
+          # Drop the version prefix ("v")
+          RELEASE_VERSION=${RELEASE_VERSION:1}
+
+          if [[ $RELEASE_TYPE == "patch" ]]
+          then
+              NEXT_VERSION=v$(./.github/workflows/util/semver bump $RELEASE_TYPE $RELEASE_VERSION)
+          elif [[ $RELEASE_TYPE == "minor" ]]
+          then 
+              NEXT_VERSION=v$(./.github/workflows/util/semver bump minor $RELEASE_VERSION)          
+          fi
+
+          echo "::set-output name=next_version::$NEXT_VERSION"
+
+      - name: Determine branch version
+        env:
+          RELEASE_VERSION: ${{ steps.release_version.outputs.release_version }}      
+        id: branch_version
+        run: echo "::set-output name=branch_version::$(echo $RELEASE_VERSION | sed 's/\.[0-9]*\+$//')"      
+
+      - name: Log information
+        run: |         
+          echo "Release version: ${{ steps.release_version.outputs.release_version }}"
+                   
+          echo "Next version: ${{ steps.next_version.outputs.next_version }}"
+
+          echo "Branch version: ${{ steps.branch_version.outputs.branch_version }}"
+
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    needs: [initialize]
+    env:
+      RELEASE_TYPE: ${{ github.event.inputs.release_type }}      
+      RELEASE_VERSION: ${{ needs.initialize.outputs.release_version }}
+      NEXT_VERSION: ${{ needs.initialize.outputs.next_version }}
+      BRANCH_VERSION: ${{ needs.initialize.outputs.branch_version }}
+      RELEASE_BRANCH: ${{ github.event.inputs.release_branch }}
+      GH_PULL_URI: https://api.github.com/repos/${{ github.event.inputs.repository }}/pulls
+      Gh_RELEASE_URI: https://api.github.com/repos/${{ github.event.inputs.repository }}/releases
+    steps:    
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.releasing_branch }} 
+        
+      - name: Configure git
+        run: |
+          git config user.email 'kiali-dev@googlegroups.com'
+          
+          git config user.name 'kiali-bot'
+
+      - name: Build Helm charts
+        run: make -e VERSION=$RELEASE_VERSION clean build-helm-charts
+
+      # Switch to `master` branch before updating docs/index.yaml
+      - name: Checkout master
+        if: ${{ github.event.inputs.release_type == 'patch'}}
+        run: git checkout master
+      
+      # Anticipated preparation of Makefile so that it contains the released
+      # version when creating the git tag.
+      #
+      # This change to the Makefile is not done for patch releases, because that
+      # would break the next minor build (remember we have already switched to master).
+      - name: Prepare Makefile
+        if: ${{ github.event.inputs.release_type != 'patch'}}
+        run: sed -i -r 's/^VERSION \?= v.*/VERSION \?= v$RELEASE_VERSION/' Makefile
```

Comment:
Doesn't `RELEASE_VERSION` already contains the `v`?

Reply:
Yes, probably this is wrong, I'll fix this soon.